### DISCUSSION TITLE
How to rename or change the icon of a device ?

### DISCUSSION THREAD

**Discussion by Belphemur** [Belphemur/SoundSwitch#702] [FAQ]
# Problem
I have two devices with the same name and SoundSwitch cannot distinguish between them.

**Comment by Belphemur** [comment]
# Solution

You can easily rename the device and then reset it up in SoundSwitch

# How
SoundSwitch > Playback Devices > Double click on the device > Rename > Save
![Animation](https://user-images.githubusercontent.com/197810/126901189-40ea8551-000f-4906-947c-db4a7bb4d539.gif)

**Reply by bigplayer-ai** [reply]
What icon size should I aim to when choosing icon for my audio devices? right now it shows very little icon on banners when switching audios.

**Reply by Belphemur** [reply]
32x32 is the right size :)
